main/mail/src/EBox/Mail.pm

```diff
@@ -737,12 +737,14 @@ sub _setDovecotConf
     my $gssapiHostname = $sysinfo->hostName() . '.' . $sysinfo->hostDomain();
 
     my $openchange = 0;
+    my $isNotificationsReady = 0;
```

Comment:
maybe $notificationsReady is a better name :) I can do the change when testing it